doc/source/whatsnew/v1.4.2.rst

```diff
@@ -31,7 +31,7 @@ Bug fixes
 ~~~~~~~~~
 - Fix some cases for subclasses that define their ``_constructor`` properties as general callables (:issue:`46018`)
 - Fixed "longtable" formatting in :meth:`.Styler.to_latex` when ``column_format`` is given in extended format (:issue:`46037`)
--
+- Fix incorrect hyperlink rendering when the url contains colon or other special characters (:issue:`46389`)
```

Comment:
I think this should be moved to 1.5.0.rst. It doesn't appear that this is a regression per se.

Reply:
this is not a regression, but a bug fix. easily backported, mind.

Reply:
@attack68 IIRC you might have added this feature in 1.4? If so, then okay to backport this fix for a new feature.

Reply:
fine to backport, @attack68 if you confirm this fixes in 1.4 as well then good to go.

Reply:
yes this fixed a 1.4.0 new feature.